docs/adopted-rules.md

```diff
@@ -0,0 +1,122 @@
+# Adopted Rules
+
+## Weapon and Armor Levels
+(*Source: [Fate Core](https://fate-srd.com/fate-core/more-examples-extras#weapon-and-armor-ratings)*)
```

Comment:
Nit: I prefer [fate condensed](https://fate-srd.com/fate-condensed/optional-rules#weapon-and-armor-ratings)'s explanation, but both are fine 🙂

Reply:
oh ty that just happened to be the page that turned up on The Search